### PR TITLE
Sync #4aec4e4

### PR DESCRIPTION
**Pull request by Jinjiang**
## Description of Problem

The diff of the original update: https://github.com/vuejs/docs-next/compare/91a4425...4aec4e4

And I list all the rest translation to do (with `<!-- TODO: translation -->` beside) which could be done later separately:
- `api/application-api.md`
- `api/built-in-components.md`
- `api/composition-api.md`
- `api/global-api.md`
- `guide/installation`
- `guide/render-function`
- `guide/ssr`
- `guide/transitions-list`
- `guide/typescript-support`
- `guide/migration/async-components`
- `guide/migration/custom-directives`
- `guide/migration/emits-option`
- `guide/migration/transition-as-root`
- `guide/migration/transition-group`
- `guide/migration/transision`
- `guide/ssr/build-config`
- `guide/ssr/getting-started`
- `guide/ssr/hydration`
- `guide/ssr/introduction`
- `guide/ssr/routing`
- `guide/ssr/server`
- `guide/ssr/structure`
- `guide/ssr/universal`
- `cookbook/automatic-global-registration-of-base-components.md`

## Additional Information

I didn't update the `yarn.lock` file.

Thanks.

### CODE REVIEW

File: src/guide/installation.md [hunk 1]
```diff
@@ -46,13 +46,27 @@ Vue.js 设计的初衷就包括可以被渐进式地采用。这意味着它可
 这些文件可以在 [unpkg](https://unpkg.com/browse/vue@next/dist/) 或者 [jsDelivr](https://cdn.jsdelivr.net/npm/vue@next/dist/) 这些 CDN 上浏览和下载。各种不同文件将在[以后解释](#对不同构建版本的解释)，但你通常需要同时下载开发环境构建版本以及生产环境构建版本。
 
 ## npm
-在用 Vue 构建大型应用时推荐使用 npm 安装<sup>[[1]](#footnote-1)</sup> 。NPM 能很好地和诸如 [Webpack](https://webpack.js.org/) 或 [Rollup](https://rollupjs.org/) 模块打包器配合使用。Vue 还提供了编写[单文件组件](../guide/single-file-component.html)的配套工具。
+
+在用 Vue 构建大型应用时推荐使用 npm 安装<sup>[[1]](#footnote-1)</sup> 。NPM 能很好地和诸如 [Webpack](https://webpack.js.org/) 或 [Rollup](https://rollupjs.org/) 模块打包器配合使用。
```
Comment (veaba) [inline src/guide/installation.md:50]:
```suggestion
在用 Vue 构建大型应用时推荐使用 npm 安装<sup>[[1]](#footnote-1)</sup> 。NPM 能很好地和诸如 [webpack](https://webpack.js.org/) 或 [Rollup](https://rollupjs.org/) 模块打包器配合使用。
```

Comment (veaba) [reply src/guide/installation.md:50]: 英文稳定 `Webpack` 又改回小写了。

File: src/guide/migration/custom-elements-interop.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ badges:
 
 ## 概览
 
-- **非兼容**：自定义元素白名单现在在模板编译期间执行，应该通过编译器选项而不是运行时配置来配置。
+- **非兼容**：检测并确定哪些标签应该被视为自定义元素的过程现在会在模板编译期间执行，且应该通过编译器选项而不是运行时配置来配置。
```
Comment (veaba) [inline src/guide/migration/custom-elements-interop.md:10]:
```suggestion
- **非兼容**：检测并确定哪些标签应该被视为自定义元素的过程，现在会在模板编译期间执行，且应该通过编译器选项而不是运行时配置来配置。
```

**Comment by veaba** [comment]
同步的内容+范围已校对没问题，这一分支，先消灭小块的 TODO。

**Review by Justineo — CHANGES_REQUESTED** [review]

**Comment by Jinjiang** [comment]
> 同步的内容+范围已校对没问题，这一分支，先消灭小块的 TODO。

是的，这边大块的翻译我都没有做，这样整个 PR 的进程会快一些，也避免大段翻译讨论出现在 sync 的 PR 里

**Review by Justineo — APPROVED** [review]

**Review by Justineo — APPROVED** [review]

**Pull request merged**